core/src/components/select/select.ionic.outline.scss

```
@@ -50,3 +50,11 @@
 
   box-sizing: border-box;
 }
+
+// Focus
+// ---------------------------------------------
+
+:host(.ion-focused.select-fill-outline) {
+  --border-color: var(--highlight-color);
+  --border-width: #{globals.$ion-border-size-050};
+}
```

Apologies, I just noticed that on the designs, only the focus has a greater width, not the valid or invalid. So let's do something like:

```suggestion
:host(.ion-focused.select-fill-outline) {
  --border-color: var(--highlight-color);
}

:host(.ion-focused.select-fill-outline:not(.ion-invalid):not(.ion-valid)) {
  --border-width: #{globals.$ion-border-size-050};
}
```

---

Thanks, no problem, I should have noticed that myself!